client/assets/components/document/document_mail/document_mail.html

```diff
@@ -1,6 +1,6 @@
 <div class="detail vertical grid-content">
   <!-- show head field linked with a url. -->
-  <h3 class="head-field" ng-if="doc.subject"><a href="#"
+  <h3 class="head-field" ng-if="doc.subject"><a href="http://asfmail.lucidworks.io/"
```

Comment:
Interesting observation!  For mail messages, parent_s will actually have the right info. I am not sure that this will work for website pages or github.